tests/workingWithAPI.spec.ts

```diff
@@ -19,27 +19,23 @@ test.beforeEach( async({page}) => {
 
 });
 
-test('mocking API request', async ({page}) => {
+test('Mocking API request', async ({page}) => {
     const pm = new PageManager(page)
     
     await pm.navigateTo().ownersPage()
 
     await pm.onOwnersPage().validateOwnerCount(2)
 
     // Extracting full name from API Mock to click on link in 
-    const testOwnerFullName = `${testOwners[0]['firstName']} ${testOwners[0]['lastName']}`    
+    const owner = testOwners.find(owner => owner.id === 1)
```

Comment:
You can add a single "!" at the end of this string, so you can later get rid of all other "question marks" and "explanation signs" to make it look nicer.
`const owner = testOwners.find(owner => owner.id === 1)!`